docs/appendix/glossary/process.md

```diff
@@ -2,14 +2,14 @@
 title: Process
 id: process
 full_link:
-short_description: Processes are binaries or scripts that run on a part.
+short_description: Processes are commands such as binaries or scripts that run on a part.
 aka:
 ---
 
-Processes are binaries or scripts that run on a {{< glossary_tooltip term_id="part" text="part" >}}.
+Processes are binaries, scripts, or other commands that run on a {{< glossary_tooltip term_id="part" text="part" >}}.
```

Comment:
As is this actually isn't correct
```suggestion
Processes are instances of binaries, scripts, or other commands that run on a {{< glossary_tooltip term_id="part" text="part" >}}.
```

Reply:
Hmm but an instance of a command (which isn't a binary or a script) isn't really how you'd word it either is it? Not sure how to word

Reply:
It's generally and instance of a program. 
```suggestion
Processes are instances of programs, invoked by commands running binaries or scripts on a {{< glossary_tooltip term_id="part" text="part" >}}.
```

Reply:
That is how we've tended to think about them but, since that's not technically the whole truth it seems the definition here should say what all they can be. But I guess since the edits to the Processes section and page say they can be anything it's okay?